project_resource_calendar/views/miscellaneous_view.xml

```diff
@@ -0,0 +1,45 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <record id="action_miscellaneous" model="ir.actions.act_window">
+        <field name="name">Miscellaneous</field>
+        <field name="res_model">resource.calendar.miscellaneous</field>
+        <field name="view_type">form</field>
+        <field name="view_mode">tree,form</field>
+    </record>
+    <menuitem id="configuration_sub_menu"
+              name="Configuration"
+              sequence="350"
+              action="action_miscellaneous"
+              parent="calendar.mail_menu_calendar"
+              groups="group_resource_calendar_manager"/>
+    <menuitem id="miscellaneous_sub_menu"
+              name="Miscellaneous"
+              parent="configuration_sub_menu"
+              action="action_miscellaneous"
+              sequence="20"
+              groups="group_resource_calendar_manager"/>
+    <record id="miscellaneous_tree" model="ir.ui.view">
```

Comment:
better resource_calendar_miscellaneous_tree (same comment for form)

Reply:
@rimbendhaou Changed